modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -118,22 +135,18 @@ The certificate must be set with a passphrase and a CN that specifies the domain
 [NOTE]
 This field is enabled after you specify an alias.
 
-... Verify you have entered values for *Keystore File*, *Keystore Passcode*, *Alias*, and *Key Passcode* before selecting other options or deploying.
+... Use the *Details* button to view the information you entered for *Keystore File*, *Keystore Passcode*, *Alias*, and *Key Passcode* before selecting other options or deploying.
 ... Select *Deploy*. The *Keystore Passcode* and *Key Passcode* fields are blanked out for security reasons.
 **** If you select a different *Alias* value, the *Key Passcode* field is again enabled.
 **** If you upload a new keystore file, the *Alias* and *Keystore Passcode* fields are again enabled and the *Alias* field contents are cleared.
 +
 The JKS file information is saved in the global secrets group for your organization.
 
-** Option 3: *Import from Secrets Manager* (for advanced users)
+** Option 3: *Import from Secrets Manager* (For Advanced Users)
```

Comment:
Consider moving the parenthetical text into the beginning of the narrative immediately following the title.
GLOBAL